frontend/packages/ux-editor/src/classes/SavableFormLayoutSettings.test.ts

```diff
@@ -0,0 +1,21 @@
+import { SavableFormLayoutSettings } from '@altinn/ux-editor/classes/SavableFormLayoutSettings';
+import { formLayoutSettingsMock } from '../testing/mocks';
+import { FormLayoutSettings } from '@altinn/ux-editor/classes/FormLayoutSettings';
+
+describe('SavableFormLayoutSettings', () => {
+  const saveFormLayoutSettings = jest.fn();
```

Comment:
Det har ikke noe å si nå som det bare er én test her, men du kan godt legge til `afterEach(jest.clearAllMocks)` her, slik at det blir skalerbart for flere tester. Eventuelt definere `saveFormLayoutSettings` i `setupLayoutSettings` og returnere den derfra.